config/metasrv.example.toml

```diff
@@ -65,6 +65,34 @@ node_max_idle_time = "24hours"
 ## The number of threads to execute the runtime for global write operations.
 #+ compact_rt_size = 4
 
+## TLS configuration for store backend (only applicable for PostgreSQL/MySQL backends)
+## When using PostgreSQL or MySQL as metadata store, you can configure TLS here
+[store_tls]
```

Comment:
I think `backend_tls` is better than `store_tls`.

Because we choose `backend` as the name of metadata store:

https://github.com/GreptimeTeam/greptimedb/blob/8639961cc98cc9e1e57772b449436d8a41a4d912/config/metasrv.example.toml#L20